### ISSUE TITLE
Livestreams

### ISSUE THREAD

**Issue by Owez** [Owez/yark#18]
Listening into livestreams which are marked as non-saved would be useful because otherwise all livestreams go unarchived. For this to work I need to rely on a yt-dlp listener which I can impl into the daemon once #12 is finished.

Livestreams which get saved can be treated as normal videos, so they won't be recorded real-time. Livestreams that go un-saved are different than normal videos, but i'm not sure if they deserve their own `"livestreams": []` yark.json section.

**Comment by Owez** [comment]
Tricky one because we now save past livestreams as of b4cbb2e8f72df6df0f334fe51519f60421791152. Might be best to put this as a wontfix, as this package is meant to archive rather than record. Maybe another tool in the future could be set up to poll youtube for ongoing livestreams in a channel and save them down to your own archive or upload them to youtube and save them back with Yark. Not sure about the YT policy on this so I wouldn't encourage it.
